README.md

```diff
@@ -63,12 +57,7 @@ WebSocket mode with a web frontend.
 
 The `run.sh` script can be used to spin up a `podman` container of the Container
 JFR Client, running alone but set up so that it is able to introspect itself
-with JFR. This can be achieved by doing `sh run.sh -it` and then typing
-`connect localhost` into the client shell that appears. When running in
-this container, all three execution modes described above are still available
-and accessible using the same mthods. Some client shell demo scripts are also
-available in the `demos` directory. These can be used with batch mode, ex.
-`sh run.sh "$(more demos/print_help)"`.
+with JFR. This can be achieved by doing `sh run.sh -it`.
```

Comment:
While we're here, let's remove that `-it`. I'd rather point users to a tool like `websocat` for use with the WebSocket mode than encourage usage of the interactive mode. Maybe we should link to `websocat`?